frontend/packages/dev-console/src/components/monitoring/alerts/__tests__/monitoring-alerts-utils.spec.tsx

```diff
@@ -16,8 +16,8 @@ describe('monitoring-alerts-utils', () => {
   });
   it('row should be collapse if rule state is not FIRING', () => {
     const alertRules = getAlertsAndRules(rules?.data).rules;
-    alertRules[0].state = RuleStates.Inactive;
+    alertRules[1].alerts[0].state = AlertStates.Pending;
```

Comment:
Can you keep the tests for both rules here?

```jsx
    alertRules[0].alerts[0].state = AlertStates.Inactive; // or what ever
    alertRules[1].alerts[0].state = AlertStates.Pending;
```

Reply:
Added the tests for other states as well.